subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/PerformanceReportScenarioTeamCityExecution.groovy

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.performance.results
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import org.gradle.performance.measure.DataSeries
+
+
+/**
+ * Each instance represents a performance execution, i.e. a row in performance execution database
```

Comment:
This comment is not true anymore, right? This represents an element in the results json produced by the performance test task, right?

Reply:
Yes, updated the comments.